media-capabilities/index.js

```diff
@@ -16,22 +16,22 @@
 
 const supported = typeof window !== 'undefined' && 'mediaCapabilities' in navigator;
 
-const useMediaCapabilities = (mediaConfig, initialMediaCapabilities = {}) => {
-  let mediaCapabilities = {
+const useMediaCapabilitiesDecodingInfo = (mediaDecodingConfig, initialMediaCapabilitiesInfo = {}) => {
+  let mediaCapabilitiesInfo = {
     supported,
-    hasMediaConfig: !!mediaConfig
+    hasMediaDecodingConfig: !!mediaDecodingConfig
```

Comment:
What's the point of this? This seems like it should just be an internal variable.

Reply:
@wingleung cc @addyosmani 

I agree with @stramel.
I guess try/catch block should wrap `navigator.mediaCapabilities.decodingInfo(mediaDecodingConfig)` for handling exceptions without `hasMediaDecodingConfig`.

A TypeError is raised if the MediaConfiguration passed to the decodingInfo() method is invalid, either because the type is not video or audio, the contentType is not a valid codec MIMME type, the media decoding configuration is not a valid value for the media decoding type, or any other error in the media configuration passed to the method, including omitting values required in the media decoding configuration. from [MediaCapabilities.decodingInfo()](https://developer.mozilla.org/en-US/docs/Web/API/MediaCapabilities/decodingInfo)